client/src/App.js

```diff
@@ -1,17 +1,22 @@
 import React from "react";
 import { MuiThemeProvider } from "@material-ui/core";
-import { BrowserRouter, Route } from "react-router-dom";
+import { BrowserRouter, Route, Switch } from "react-router-dom";
 
 import { theme } from "./themes/theme";
 import LandingPage from "./pages/Landing";
+import LoginPage from "./pages/Login";
+import SignUpPage from "./pages/Signup";
 
 import "./App.css";
 
 function App() {
   return (
     <MuiThemeProvider theme={theme}>
       <BrowserRouter>
-        <Route path="/" component={LandingPage} />
```

Comment:
you should add a navbar component here that exists in all components - it'll have different rendering though depending on props.location

Reply:
@skassam21 I was going to do this but I decided to use Material-UI's App Bar instead since it seemed more flexible. I also just put it directly into the profile page code since it's a single page. Should I not do it this way?